build.sbt

```diff
@@ -4,9 +4,9 @@ description := "sbt web support"
 
 libraryDependencies ++= Seq(
   "com.typesafe.akka" %% "akka-actor" % "2.3.9",
-  "org.webjars" % "webjars-locator" % "0.21",
+  "org.webjars" % "webjars-locator" % "0.23",
```

Comment:
Yes, I do. But the tests fail with it and I'm not sure why. Would you be able to take a stab at getting it to 0.24? See my comment on the main page of the PR for the error message I'm getting